db/migrate/20191125160749_create_users.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :email
+      t.string :username
+      t.boolean :activated
+      t.boolean :admin
```

Comment:
I'm curious about these two attributes:
* Did you add them?
* Are you using them?
* Are they required?

Reply:
yes, no and  no. we are going to delete them.